eclair-core/src/test/scala/fr/acinq/eclair/payment/MultiPartPaymentLifecycleSpec.scala

```diff
@@ -384,6 +384,20 @@ class MultiPartPaymentLifecycleSpec extends TestKitBaseClass with FixtureAnyFunS
     assert(result.failures.length === 1)
   }
 
+  test("abort if payment gets confirmed on chain") { f =>
```

Comment:
nit: `confirmed` implies success in my mind, whereas most of the time it will be an htlc-timeout that will happen on-chain when we're sending a payment, so `settled` feels better.

```suggestion
  test("abort if payment gets settled on chain") { f =>
```